src/onepirate/modules/form/CheckBox.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import FormGroup from '@material-ui/core/FormGroup';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Checkbox from '@material-ui/core/Checkbox';
+import Grid from '@material-ui/core/Grid';
+
+export default function CheckboxLabels() {
```

Comment:
Would radio buttons be better as they are mutually exclusive 😮 I can select both "Researcher" and "Volunteer" in the form but I think they should be mutually exclusive: 
![image](https://user-images.githubusercontent.com/43816143/86568464-7d3bd580-bf9f-11ea-82d2-dd893de2c5f2.png)

Reply:
I think radio buttons are better too, thanks for the suggestion! Made the change already.